storage/rocksdb/rdb_native_dd.cc

```diff
@@ -70,8 +70,9 @@ bool rocksdb_dict_set_server_version() {
       ->set_server_version();
 }
 
-bool rocksdb_is_supported_system_table(const char *, const char *, bool) {
-  return false;
+bool rocksdb_is_supported_system_table(const char *, const char *,
```

Comment:
This function, unfortunately, has to go back to `ha_rocksdb.cc`, because it's not related to DD tables at all